### PR TITLE
TOAZ-303: Update Leo to pass pet UAMI objectId to cromwell chart

### PR DESCRIPTION
**Pull request by rtitle**
Ticket: https://broadworkbench.atlassian.net/browse/TOAZ-303

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala [hunk 1]
```diff
@@ -127,6 +127,7 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
       "config.batchAccountName=batch," +
       "config.batchNodesSubnetId=subnet1," +
       s"config.drsUrl=${ConfigReader.appConfig.drs.url}," +
+      "config.workflowExecutionIdentity=identity-id" +
```
Comment (Qi77Qi) [inline http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala]: missing `,`?

Comment (rtitle) [reply http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala]: Yep, fixed

**Review by Qi77Qi — APPROVED** [review]

**Review by jordansafer — APPROVED** [review]

**Comment by Qi77Qi** [comment]
another instance of
```
      java.lang.RuntimeException: terra-quality-2ea79b77/automation-test-aooemvf0z errored due to List(RuntimeError(Required 'compute.images.get' permission for 'projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-2-0-51-debian10-a46b242',Some(7),2023-01-03T22:32:32Z,Some(TraceId(cb5c96ba12194f84b74ca050d79b7f65/3939911508519804487))))
```

https://console.cloud.google.com/logs/query;query=%22cb5c96ba12194f84b74ca050d79b7f65%2F3939911508519804487%22;timeRange=2023-01-03T22:02:00.000Z%2F2023-01-03T23:01:00.000Z;summaryFields=jsonPayload%252FgoogleCall:false:32:beginning;cursorTimestamp=2023-01-03T22:31:47.355874621Z?referrer=search&authuser=0&project=broad-dsde-qa ...looks slightly different from what I fixed in https://github.com/DataBiosphere/leonardo/pull/3082

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Pull request merged**